File: Programming Basics with Python/03 Conditional statements advanced/More exercises/06_Truck_Driver.py
season = input().lower()
km_per_month = float(input())
price_per_km = 0
if km_per_month <= 5000:
    if season == "summer":
        price_per_km = 0.9
    elif season == "winter":
        price_per_km = 1.05
    else:
        price_per_km = 0.75
elif 5000 < km_per_month <= 10000:
    if season == "summer":
        price_per_km = 1.1
    elif season == "winter":
        price_per_km = 1.25
    else:
        price_per_km = 0.95
else:
    price_per_km = 1.45
profit = 4 * km_per_month * price_per_km
taxes = profit * 0.1
result = profit - taxes
print(f"{result:.2f}")

File: JS Basics/03 Nested Conditional Statements - More Exercises/06. Truck Driver.js
function driver(season, km) {
    km = Number(km);
    let profit = 1.45 * km
    if (season == "Spring" || season == "Autumn") {
        if (km <= 5000) {
            profit = 0.75 * km;
        }
        else if (5000 < km && km <= 10000) {
            profit = 0.95 * km;
        }
    }
    else if (season == "Summer") {
        if (km <= 5000) {
            profit = 0.9 * km;
        }
        else if (5000 < km && km <= 10000) {
            profit = 1.1 * km;
        }
    }
    else if (season == "Winter") {
        if (km <= 5000) {
            profit = 1.05 * km;
        }
        else if (5000 < km && km <= 10000) {
            profit = 1.25 * km;
        }
    }
    profit *= 0.9 * 4;
    console.log(profit.toFixed(2))
}
driver("Autumn", 8600)